package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "getnyumba_node",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Add a script for running the application I suggest running using `nodemon` in our dev dependencies https://www.npmjs.com/package/nodemon